.github/workflows/publish-java-apidocs.yml

```diff
@@ -0,0 +1,48 @@
+name: Update Java API Docs
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Generate Java docs
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Is this specifying a docker container which has the ORT source and can be used to build the javadoc? If so where is `ubuntu-latest` defined?

Reply:
ubuntu-latest is the container that the GitHUb actiions runs in. The ort source is checked out separately